samples/addons/extras/zipkin.yaml

```diff
@@ -22,6 +22,12 @@ spec:
           env:
             - name: STORAGE_METHOD
               value: "mem"
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: 9411
+            initialDelaySeconds: 5
+            periodSeconds: 30
```

Comment:
Should we shorten the period seconds? 30s seems to be pretty long.

Reply:
Maybe. This was mostly a super conservative attempt to see if it would help.

Maybe:

```
periodSeconds: 5
failureThreshold: 10
```

?

Reply:
yep, looks good.

Reply:
I think the first probe is sent at initialDelaySeconds+periodSeconds if I recall correctly. So 5s sounds better to me